pyproject.toml

```diff
@@ -22,6 +22,7 @@ classifiers = [
     "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
 ]
 dependencies = [
+    'bohrium-sdk',
```

Comment:
dpdispatcher is packaged in conda-forge. Please submit this new package into conda-forge, or make it optional.